src/foam/nanos/auth/User.js

```diff
@@ -290,6 +290,7 @@ foam.CLASS({
     {
       class: 'Boolean',
       name: 'system',
+      value: false,
```

Comment:
We already have the 'type' property. Couldn't we just add 'system' as a new type?

Reply:
We plan to sub-class User into these types such as BusinessUser so the type as it exists now will go away.